mantle/kola/tests/ignition/luks.go

```diff
@@ -50,6 +50,17 @@ func init() {
 		ExcludeArchitectures: []string{"s390x"}, // no TPM backend support for s390x
 		Tags:                 []string{"luks", "tpm", "tang", "sss", kola.NeedsInternetTag, "reprovision"},
 	})
+	register.RegisterTest(&register.Test{
+		Run:                  luksCEXTest,
+		ClusterSize:          0,
+		Name:                 `luks.cex`,
+		Description:          "Verify that the rootfs is encrypted with CEX.",
+		Flags:                []register.Flag{},
+		Distros:              []string{"rhcos"},
+		Platforms:            []string{"qemu"},
+		ExcludeArchitectures: []string{"aarch64", "ppc64le", "x86_64"},
+		Tags:                 []string{"luks", "cex", kola.NeedsInternetTag, "reprovision"},
```

Comment:
I don't think this test needs Internet access, right?

Reply:
Yes, Internet access is not required.